operations/vouchercodes.md

```diff
@@ -127,50 +116,65 @@ Adds voucher codes to the specified [Vouchers](vouchers.md#voucher). Note this o
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../guidelines/multi-property.md), ignored otherwise. |
-| `VoucherCodeParameters` | array of [Voucher code parameters](#voucher-code-parameters) | required, max 1000 items | Voucher codes to be added. |
+| `EnterpriseId` | string | optional | Unique identifier of the enterprise. Required when using [Portfolio Access Tokens](../guidelines/multi-property.md), ignored otherwise. |
+| `VoucherCodeParameters` | array of [Voucher Code Parameters](vouchercodes.md#voucher-code-parameters) | required, max 1000 items | Voucher codes to be added. |
 
-#### Voucher code parameters
+#### Voucher Code Parameters
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `VoucherId` | string | required | Unique identifier of [Voucher](#voucher) the code belongs to. |
-| `Value` | string | required, max 128 characters | Value of voucher code used by customers. |
+| `Value` | string | required, max length 128 characters | Value of voucher code used by customers. |
 | `ValidityStartUtc` | string | optional | If specified, marks the beginning of interval in which the code can be used. |
 | `ValidityEndUtc` | string | optional | If specified, marks the end of interval in which the code can be used. |
 
 ### Response
 
 ```javascript
 {
-    "VoucherCodes": [
-        {
-            "Id": "8c364829-c7ae-4972-b67f-93ea704d7228",
-            "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
-            "Value": "0E5856B0A73E62B7E446",
-            "ValidityStartUtc": null,
-            "ValidityEndUtc": null,
-            "CreatedUtc": "2023-10-30T13:37:16Z",
-            "UpdatedUtc": "2023-10-30T13:37:16Z",
-            "ActivityState": "Active"
-        },
-        {
-            "Id": "dc9d3488-7fc4-4fc9-a524-14e6504d8734",
-            "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
-            "Value": "021D047E42A5FD522CBA",
-            "ValidityStartUtc": "2023-10-09T22:00:00Z",
-            "ValidityEndUtc": "2023-10-09T22:00:00Z",
-            "CreatedUtc": "2023-10-09T15:08:14Z",
-            "UpdatedUtc": "2023-10-09T15:08:14Z",
-            "ActivityState": "Active"
-        }
-    ]
+  "VoucherCodes": [
```

Comment:
Just a note to make sure we are consistent with tab spacing. Presumably this is some parameter in the Generator code, so it will be consistent for all auto-generated content. But I note that it is based on 2 spaces, whereas the old manually created docs uses 4 spaces. Either is fine from my point of view, as long as it is a conscious decision.